packages/fluentui/react-northstar-emotion-renderer/src/focusVisiblePlugin.ts

```diff
@@ -0,0 +1,17 @@
+const notFocusVisibleRgxp = /:not\(:focus-visible\)/g;
+const focusVisibleRgxp = /:focus-visible/g;
+const trim = (selector: string) => selector.trim().replace(/\s+/g, ' ');
```

Comment:
nit: this really isn't really `trim` since it deals with whitespace inside the string. I couldn't find a decent name for this common operation on the internet... mb you'll have better luck

```suggestion
const toSingleWhitespace = (selector: string) => selector.trim().replace(/\s+/g, ' ');
```

Reply:
Updated